Sources/ProjectSpec/TargetScheme.swift

```diff
@@ -22,6 +23,9 @@ public struct TargetScheme: Equatable {
     public var environmentVariables: [XCScheme.EnvironmentVariable]
     public var preActions: [Scheme.ExecutionAction]
     public var postActions: [Scheme.ExecutionAction]
+    public var shared: Bool
+    public var orderHint: Int?
+    public var isShown: Bool?
```

Comment:
@wendyliga what was the reason again why we gave this struct 3 separate properties and `Scheme` a `management: Management` property? I think the api would be more straight forward if we keep this consistent?

Reply:
if I remember it correctly, I think it because of the `Scheme`  will create a directory called management, and for `TargetScheme`, it will only add new properties to existing file.

Reply:
I don't think that is the case. Schemes should be written into different folders depending whether or not a scheme should be shared. 

Shared schemes should be written to: `*.xcodeproj/xcshareddata/xcschemes`
Non shared / personal schemes to: `*.xcodeproj/xcuserdata/<user-name>.xcuserdatad/xcschemes`

The plist with the management info should always be in: `*.xcodeproj/xcuserdata/<user-name>.xcuserdatad/xcschemes/xcschememanagement.plist`

So if we want to support the `shared` boolean option properly, we should move the schemes from the shared schemes folder to the personal one if it is set....  I'll see if I can add support for this in XcodeProj first.

Reply:
Maybe it would be better to just remove the `shared: Bool` option for now? Since we're generating the schemes I wonder if setting the `shared` option would be used a lot since it would only determine where the generated scheme is going to be placed (`xcshareddata` or `xcuserdata`). I guess it would only make sense if you use XcodeGen to generate a project once that you're then going to check into git? 

Would you agree that it's fine to remove the option for now until there's native support for `xcuserdata` schemes at  XcodeProj? The alternatives are either waiting on XcodeProj support, or manually moving the generated schemes from xcshareddata if the `shared` option is set to `false`.

Reply:
Maybe it would be better to just remove the `shared: Bool` option for now? Since we're generating the schemes I wonder if setting the `shared` option would be used a lot since it would only determine where the generated scheme is going to be placed (`xcshareddata` or `xcuserdata`). I guess it would only make sense if you use XcodeGen to generate a project once that you're then going to check into git? 

@wendyliga @yonaskolb would you agree that it's fine to not add the `shared` option for now until there's native support for `xcuserdata` schemes at  XcodeProj? The alternatives are either waiting on XcodeProj support, or manually moving the generated schemes from xcshareddata if the `shared` option is set to `false`.

Reply:
This is now done by XcodeProj \0/